src/Stripe.net/Services/Customers/CustomerService.cs

```diff
@@ -36,6 +36,16 @@ public virtual Task<Customer> CreateAsync(CustomerCreateOptions options, Request
             return this.CreateEntityAsync(options, requestOptions, cancellationToken);
         }
 
+        public virtual FundingInstructions CreateFundingInstructions(string id, CustomerCreateFundingInstructionsOptions options = null, RequestOptions requestOptions = null)
```

Comment:
We need to re-add the FundingInstructions type.

Reply:
All fixed!